Framework/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0.4</string>
+	<string>1.0.6</string>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
```

Comment:
should this one also be updated?

Reply:
No, I think.
I grepped the word. It will be _Build_ number.
_Version_ number is enough to bump the lib version.